nuget/framework/nunit.nuspec

```diff
@@ -28,32 +28,33 @@ Supported platforms:
     <tags>nunit test testing tdd framework fluent assert theory plugin addin</tags>
     <copyright>Copyright (c) 2017 Charlie Poole</copyright>
     <dependencies>
+      <group targetFramework="netstandard1.3">
+        <dependency id="NETStandard.Library" version="1.6.0" />
+        <dependency id="System.Threading.Thread" version="4.3.0" />
+      </group>
       <group targetFramework="netstandard1.6">
         <dependency id="NETStandard.Library" version="1.6.0" />
-        <dependency id="System.Runtime.Loader" version="4.0.0" />
+        <dependency id="System.Runtime.Loader" version="4.3.0" />
+        <dependency id="System.Threading.Thread" version="4.3.0" />
       </group>
     </dependencies>
   </metadata>
   <files>
     <file src="LICENSE.txt" />
     <file src="NOTICES.txt" />
     <file src="CHANGES.md" />
-    <file src="bin/net-2.0/nunit.framework.dll" target="lib/net20" />
-    <file src="bin/net-2.0/nunit.framework.xml" target="lib/net20" />
-    <file src="bin/net-2.0/NUnit.System.Linq.dll" target="lib/net20" />
-    <file src="bin/net-3.5/nunit.framework.dll" target="lib/net35" />
-    <file src="bin/net-3.5/nunit.framework.xml" target="lib/net35" />
-    <file src="bin/net-4.0/nunit.framework.dll" target="lib/net40" />
-    <file src="bin/net-4.0/nunit.framework.xml" target="lib/net40" />
-    <file src="bin/net-4.5/nunit.framework.dll" target="lib/net45" />
-    <file src="bin/net-4.5/nunit.framework.xml" target="lib/net45" />
-    <file src="bin/netstandard16/nunit.framework.dll" target="lib/netstandard1.6" />
-    <file src="bin/netstandard16/nunit.framework.xml" target="lib/netstandard1.6" />
-    <file src="bin/portable/nunit.framework.dll" target="lib/portable-net45+win8+wp8+wpa81" />
-    <file src="bin/portable/nunit.framework.xml" target="lib/portable-net45+win8+wp8+wpa81" />
-    <file src="bin/portable/nunit.framework.dll" target="lib/MonoAndroid" />
-    <file src="bin/portable/nunit.framework.xml" target="lib/MonoAndroid" />
-    <file src="bin/portable/nunit.framework.dll" target="lib/Xamarin.iOS10" />
-    <file src="bin/portable/nunit.framework.xml" target="lib/Xamarin.iOS10" />
+    <file src="bin/net-2.0/nunit.framework.dll" target="lib\net20" />
```

Comment:
That's very odd. I would have thought forward slashes would be preferred for the new, x-plat NuGet!